opencti-platform/opencti-front/src/components/list_lines/ListLines.js

```diff
@@ -271,6 +272,48 @@ class ListLines extends Component {
                 {t('entitie(s)')}
               </div>
             )}
+            {(exportEntityType === 'Report') && redirectionMode && (
```

Comment:
redirection mode is enough, you shouldn't have it available if you are not in a report view

Reply:
fixed